packages/matter.js/src/cluster/GeneralCommissioningCluster.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2022 Project CHIP Authors
+ * Copyright 2022-2023 Project CHIP Authors
```

Comment:
These one line copyright updates aren't strictly necessary, but they don't hurt either.  Is the linter mandating this, or is it just precision?

Reply:
just precision ... Having such an update once a year is ok I think ... or we remove it, but having a mixure of different year infos might be strange